src/index.js

```diff
@@ -1,27 +1,31 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
-import HomePage from './pages/HomePage.js';
+import AboutPage from './pages/AboutPage.js';
 import ResourcePage from './pages/ResourcePage.js';
 import XpPage from './pages/XpPage.js';
 import SosPage from './pages/SosPage.js';
 import SignUpPage from './pages/SignUpPage.js';
 import LogInPage from './pages/LogInPage.js';
 import ProfilePage from './pages/ProfilePage.js';
+import BlogPage from './pages/BlogPage.js';
+import SplashPage from './pages/SplashPage.js';
```

Comment:
this is a mess, would it be possible for us to do like an entrypoint to pages?
the entrypoint has all the pages defined as exports and then in index we just import the entrypoint? 

- it's possible that this is a lil complex for the structure we think we'll end up with

Reply:
I dont know what you mean by entrypoint. If we make one page with all the "page" components exported, that page will be even more messy. 

More importantly, this is an index page. I'm sorry you find it "messy" (I dont), but the purpose is to import the page and then route to them. It only serves one purpose, and that purpose is very clear, and extremely standard for an `index` file. I think it is better to have everything clearly stated and straight forward. 

Also, this page is really only 33 lines - how can it be that bad when its not even a full screen length? Having a lot of imports doesnt automatically make a file problematic. This is also ALL of the pages we will ever have to route to, so this page is basically complete. 

I disagree that there is a problem.